tools/packages/mini-parse/src/Stream.ts

```diff
@@ -17,7 +17,7 @@ export interface Stream<T extends Token> {
 
   /** src text
    *
-   * TODO: Remove this (move it into an extended stream type)
+   * LATER Remove this (move it into an extended stream type)
```

Comment:
I'll have to look at this again whenever I get around to messing around with mini-parse.